tools/plotly_ml_performance_plots/plot_ml_performance.py

```diff
@@ -140,7 +140,7 @@ def main(infile_input, infile_output, infile_trained_model):
         plotly.offline.plot(fig_roc, filename="output_roc.html", auto_open=False)
 
     except Exception as exp:
-        pass
+        print("Plotting the ROC-AUC graph failed. This exception was raised: {}".format(exp))
```

Comment:
I discovered the reason that the reason the ROC-AUC curves were not being created: an error was being silently caught here. I added this line to make debugging easier in the future.